src/sample_data/valid/Entity-emsl-example-00002.json

```diff
@@ -0,0 +1,1037 @@
+[
+  {
+    "id": "UUID:13eee540-e113-11eb-ba80-0242ac130004",
+    "name": "WT_0.5%G_light",
+    "ber_data_source": "EMSL",
+    "entity_type": ["sample"],
+    "uri": "https://sc.emsl.pnnl.gov/?subSrc=data%2F%3FprojectId%3D49960",
+    "properties": [
+      {
+        "attribute": {
+          "label": "data avilable"
+        },
+        "raw_value": "imaging- electron"
+      },
+      {
+        "attribute": {
+          "label": "sample type"
+        },
+        "raw_value": "pure_culture"
+      },
+      {
+        "attribute": {
+          "id": "MIXS:0001043",
+          "label": "growth faility"
+        },
+        "raw_value": "incubation_in_lab"
+      },
+      {
+        "attribute": {
+          "id": "MIXS:0000011",
+          "label": "collection date"
+        },
+        "raw_value": "2021-07-12"
+      },
+      {
+        "attribute": {
+          "id": "MIXS:0000016",
+          "label": "sample material processing"
+        },
+        "raw_value": "high pressure flash freeze"
+      },
+      {
+        "attribute": {
+          "id": "MIXS:0000327",
+          "label": "storage conditions"
+        },
+        "raw_value": "frozen"
+      },
+      {
+        "attribute": {
+          "id": "MIXS:0000110",
+          "label": "sample storage temperature"
+        },
+        "raw_value": "-80 degree Celsius"
+      },
+      {
+        "attribute": {
+          "label": "shipped sample size"
+        },
+        "raw_value": "50 mL x OD750=6.5"
+      },
+      {
+        "attribute": {
+          "label": "growth_medium"
+        },
+        "raw_value": "0.5% glucose broth"
+      },
+      {
+        "attribute": {
+          "label": "treatment"
+        },
+        "raw_value": "70-80 umol/m2s white light"
+      },
+      {
+        "attribute": {
+          "label": "treatment"
+        },
+        "raw_value": "mixotrophic growth"
+      },
+      {
+        "attribute": {
+          "id": "MIXS:0000753",
+          "label": "oxygenation status of sample"
+        },
+        "raw_value": "aerobic"
+      },
+      {
+        "attribute": {
+          "id": "MIXS:0000028",
+          "label": "observed biotic relationship"
+        },
+        "raw_value": "free living"
+      },
+      {
+        "attribute": {
+          "id": "MIXS:0000032",
+          "label": "trophic level"
+        },
+        "raw_value": "mixotroph"
+      },
+      {
+        "attribute": {
+          "id": "MIXS:0000029",
+          "label": "host scientific name"
+        },
+        "raw_value": "3075"
```

Comment:
this should be an ID
NCBI:txid3075

how are we formatting IDs?
@cmungall @ialarmedalien

Reply:
```suggestion
        "raw_value": "NCBITaxon:3075"
```